sdk/eventhub/Azure.Messaging.EventHubs.Processor/src/EventProcessorClient.cs

```diff
@@ -1143,9 +1141,9 @@ private async Task RunAsync(CancellationToken cancellationToken)
         {
             cancellationToken.ThrowIfCancellationRequested<TaskCanceledException>();
 
-            if (ActivePartitionProcessors.TryRemove(partitionId, out var t))
+            if (ActivePartitionProcessors.TryRemove(partitionId, out var activeTokenSource))
```

Comment:
If we rename it above, we should rename it here as well.

Reply:
Agreed.